crates/bcr-ebill-wasm/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub async fn initialize_api(
     let keys = db.identity_store.get_or_create_key_pair().await?;
 
     info!("Local node id: {:?}", keys.get_public_key());
-    info!("Local npub: {:?}", keys.get_nostr_npub()?);
```

Comment:
maybe it would make sense to use `tokio_with_wasm` for spawning the nostr consumer and jobs as well?
I don't think we need 2 ways to do it in the codebase. Then we could remove wasm_bindgen_futures.

Reply:
Agreed, so `tokio_with_wasm` worked out of the box first try, that is the reason I kept it. I did not want to completely rework all the other places with this feature.

Reply:
Ah, I think we still might need the wasm_bindgen_futures dependency to be able to use async functions from JS, but at least replacing the `spawn_local` calls with this would be good :+1: